### ISSUE TITLE
Missing GCC5 support

### ISSUE THREAD

**Issue by telphan** [embeddedarm/u-boot#2]
Trying to compile this u-boot on master under GCC 5.2 will yield the following compile error:

`include/linux/compiler-gcc.h:114:30: fatal error: linux/compiler-gcc5.h: No such file or directory compilation terminated.`

Copying the file from the linux tree will yield other compile issues.

**Comment by markfeathers** [comment]
I'll look into pulling in that file, but I'm curious about that referenced issue:
+# Because current revision of u-boot-ts and linux-pxa16 do not have support for GCC5
Is this for the TS-4900 support?  The linux-pxa16 tree is for the marvell PXA168 which doesn't use u-boot.  This is the git for the imx6 based boards:
https://github.com/embeddedarm/linux-3.10.17-imx6

This has a 3.14 branch in it if you wanted a more recent kernel, and it looks like it already has the gcc5 support.

**Comment by telphan** [comment]
The u-boot-ts is for the ts4900 support. The linux-pxa16 is for ts7700 support, indeed this board does not make use of u-boot, so they are two different packages for two different boards that do not get compiled under gcc5. 
Also a simple pull of the file will not suffice. I have tried making these packages compile under gcc5 and after about 4 compile errors that I have solved in a fairly easy manner there were still more to come, so I have just downgraded the GCC version to keep things neat in our repo.
